### PR TITLE
Bump @matrix-org/react-sdk-module-api to 0.0.4

### PR DESCRIPTION
**Pull request by andybalaam**
This gives access to the new widget customisations e.g. https://github.com/matrix-org/matrix-react-sdk/pull/10121

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by maheichyk** [comment]
@andybalaam it seems that the pipeline should be fine if restarted, so maybe could try to merge

**Review by t3chguy — CHANGES_REQUESTED** [review]
Seems like no bumping actually happened given no yarn.lock change

**Comment by andybalaam** [comment]
Good point @t3chguy . After much fighting with `yarn install` trying to prevent the wrong matrix-events-sdk version re-appearing, I realised I needed https://github.com/vector-im/element-web/pull/24708 first, then I can fix this one.

File: yarn.lock [hunk 1]
```diff
@@ -8110,7 +8103,7 @@ matrix-mock-request@^2.5.0:
 
 "matrix-react-sdk@github:matrix-org/matrix-react-sdk#develop":
   version "3.67.0"
-  resolved "https://codeload.github.com/matrix-org/matrix-react-sdk/tar.gz/da2fcdd3462bfcc3ee0033f2d5303942521e1df5"
+  resolved "https://codeload.github.com/matrix-org/matrix-react-sdk/tar.gz/4f1e5a71dec65258f5295f7d60d2e733391d304c"
```
Comment (t3chguy) [inline yarn.lock:8106]: This looks wrong.

**Review by t3chguy — CHANGES_REQUESTED** [review]

**Review by t3chguy — APPROVED** [review]

**Comment by andybalaam** [comment]
Still getting this, something is still not right with the lockfile:

```
[current branch] Install Dependencies
  Installing using yarn --frozen-lockfile
  /usr/local/bin/yarn --frozen-lockfile
  yarn install v1.22.19
  [1/4] Resolving packages...
  error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
  info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
  Error: The process '/usr/local/bin/yarn' failed with exit code 1
```

**Comment by andybalaam** [comment]
I can't understand why I'm still getting:

```
/usr/local/bin/yarn --frozen-lockfile
  yarn install v1.22.19
  [1/4] Resolving packages...
  error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
```

After my latest commit did a `yarn install` and that command works for me locally.

**Comment by andybalaam** [comment]
Ah, I think it's running against a different commit so it can compare with this one.

Given that the main Static Analysis action passed, I think this PR is fine. Merging and keeping an eye in case I'm wrong.

**Pull request merged**